meta-iot2000-example/README.SWUPDATE.md

```diff
@@ -0,0 +1,35 @@
+# Example for efibootguard + SWUpdate support with meta-iot2000
+
+The [kas](https://pypi.python.org/pypi/kas) recipes in
+
+```
+meta-iot2000/meta-iot2000-example/kas-update{,-rt}.yml
+```
+
+build the base image as well as an update artifact without and with a
+real-time kernel respectively, ready to be fed to `SWUpdate`. The recipes contain all needed layer
+references.
+
+For `SWUpdate` integration, the official [meta-swupdate](https://github.com/sbabic/meta-swupdate)
+layer is used. Per default it installs an init script to start up a webserver for update purposes,
+which is not needed for manual updates. For this purpose, it is replaced by an empty
+init script.
+
+This example of `SWUpdate` and `efibootguard` integration provides manual update capability only.
+
+
+The manual update process means, that the `swu` file must be copied onto the finished SD-card. To use it with swupdate, the following command is used:
+
+```
+swupdate -i <filepath>.swu
+```
+
```

Comment:
Then I need ```reboot``` (swupdate tells me, but better state this again). But how do I declare the update successful after the reboot? Without that, I will fall back into v1 again.